docs/modules/deployment/pages/serverless-faas.adoc

```diff
@@ -7,4 +7,173 @@ include::ROOT:partial$version-check.adoc[]
+
+Because each application is different, we can't provide any recommendations. Monitor your application and tune the memory allocation until the performance is satisfactory for your needs.   
+====
+
+Cerbos provides two deployment options for AWS Lambda:
+
+=== Cerbos Lambda Function
+
+Deploy Cerbos as a standalone AWS Lambda function that handles authorization requests directly. This deployment pattern is suitable for centralized authorization services.
+
+Download the specially-built `cerbosfunc` binary from the link:{app-github-releases-page}[releases page], which can be deployed as either a Zip package or a container image. 
+
+.SAM template for Lambda Function deployment
+
+[source,yaml,linenums]
+----
+---
+
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: Cerbos Lambda Function
+
+Globals:
+  Function:
+    Timeout: 5
+
+Resources:
+  CerbosFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      Runtime: provided.al2
+      CodeUri: dist/
+      Handler: bootstrap <1> 
+      Architectures:
+        - arm64
+      MemorySize: 1024
+      Events:
+        CheckResources:
+          Type: HttpApi
+          Properties:
+            Path: /api/check/resources
+            Method: POST
+        PlanResources:
+          Type: HttpApi
+          Properties:
+            Path: /api/plan/resources
+            Method: POST
+        HealthCheck:
+          Type: HttpApi
+          Properties:
+            Path: /
+            Method: GET
+      Environment:
+        Variables:
+          CERBOS_LOG_LEVEL: debug
+          XDG_CACHE_HOME: /tmp
+          CERBOS_CONFIG: /.cerbos.yaml
+
+Outputs:
+  CerbosFunctionAPI:
+    Description: "API Gateway endpoint URL for Cerbos Function"
+    Value: !Sub "https://${ServerlessHttpApi}.execute-api.${AWS::Region}.amazonaws.com"
+  CerbosFunction:
+    Description: "Cerbos Lambda Function ARN"
+    Value: !GetAtt CerbosFunction.Arn
+  CerbosFunctionIamRole:
+    Description: "IAM Role created for the Cerbos Lambda function"
+    Value: !GetAtt CerbosFunctionRole.Arn
+----
+<1> The `cerbosfunc` binary should be renamed to (or hard-linked as) `bootstrap` as per AWS Lambda guidelines for Go binaries.
+
+=== Cerbos Lambda Extension (Layer)
+
+Deploy Cerbos as a Lambda extension layer that runs alongside your application function. This deployment pattern is similar to a sidecar deployment.
+
+Download the specially-built `cerbosext` binary from the link:{app-github-releases-page}[releases page], which can be deployed as a Zip layer.
+The layer then can be attached to your AWS Lambda function, if it is deployed as a Zip package.
```

Comment:
Give more information on how this can be done or at least link to the relevant AWS documentation.